GTBitsOfGood/canine-assistants
# Increase Log Tags Margin

## Increase Log Tags Margin

Issue Number(s): #205

Increases the gap between the log tags in the TagDisplay component to match the Figma.

### Checklist


### How To Test

Go to the Logs tab on an individual dog page, create a log if one doesn't already exist, and ensure that the tags are spaced correctly.
